src/common/TALENTS/shaman.ts

```diff
@@ -1244,7 +1244,7 @@ const talents = createTalentList({
     manaCost: 750,
   },
   STORMBLAST_TALENT: {
-    id: 319930,
+    id: 390287,
```

Comment:
@Seriousnes this one needs to be addressed to prevent your Stormblast-related code from breaking the next time talent data is imported (which will likely be 10.1.5)

Reply:
Sorry I missed this one.